src/devices/Ds1307/Ds1307.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ds1307
+{
+    /// <summary>
+    /// Realtime Clock DS1307
+    /// </summary>
+    public class Ds1307 : IDisposable
+    {
+        /// <summary>
+        /// DS1307 Default I2C Address
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x68;
+
+        private I2cDevice _sensor = null;
+
+        /// <summary>
+        /// DS1307 DateTime
+        /// </summary>
+        public DateTime DateTime { get => ReadTime(); set => SetTime(value); }
```

Comment:
Any particular reason why you went with a property instead of just making your Read and Set time methods public? I don't have a strong preference on any, I just think that given you are actually executing some code that might throw when reading or writing bytes, it might be more intuitive to have this be a method.

Reply:
> I don't have a strong preference on any

Me too. My first `iot` submission was using the public method, but some members prevented me from doing so, haha. Now, using properties is a feature of `iot` 😄

Reply:
I see, that's fine then, let's keep it as a property for now.